tests/pytorch/mock_sparse2/test_elementwise_op.py

```diff
@@ -0,0 +1,115 @@
+import sys
+
+import backend as F
+import pytest
+import torch
+
+from dgl.mock_sparse2 import (add, create_from_coo, create_from_csc,
+                              create_from_csr, diag)
+
+# TODO(#4818): Skipping tests on win.
+if not sys.platform.startswith("linux"):
+    pytest.skip("skipping tests on win", allow_module_level=True)
+
+
+@pytest.mark.parametrize("val_shape", [(), (2,)])
```

Comment:
Could you check whether autograd is working? I am not sure whether `sparse_coo_tensor` has autograd.

Reply:
It's not. I've added an item in #5022 .